canonical-web-and-design/react-components
# Migrate Accordion and AccordionSection to TypeScript

## Done
* Migrate `Accordion` and `AccordionSection` to TypeScript.
* Add base types in `types`.

## QA
* Ensure storybook renders correctly.
* Build and pack, `yarn build; npm pack`, install tarball with `yarn add` in `maas-ui` to ensure `maas-ui` builds.
